src/test/java/racingcar/RacingCarTest.java

```diff
@@ -0,0 +1,83 @@
+package racingcar;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class RacingCarTest {
+    private Car car;
+
+    @BeforeEach
+    void setUp() {
+        car = new Car();
+    }
+
+    @Test
+    @DisplayName("자동차의 초기 위치는 0이다")
+    void returnZero_WhenInitialized() {
+        assertThat(car.getPosition()).isEqualTo(0);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {4, 5, 6, 7, 8, 9})
```

Comment:
```suggestion
    @ValueSource(ints = {4, 9})
```
강의에서도 다룬 내용임
테스트 데이터가 무조건 많다고 좋은 것음 아님
가능하면 경계값에 해당하는 값만으로 테스트할 것을 추천